config/prow/cluster/k8sgsmtools-rbac.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: default
+  name: "ks8-gsm-tools"
```

`find . -name '*.yaml' -exec sed -i -e s/"k8s-gsm-tools"/k8s-gsm-tools/g` {} \;`
```suggestion
  name: ks8-gsm-tools
```

---

Setup a workload identity annotation on this as if you had one, and I'll setup a GCP SA and binding etc. prior to merging this

eg: 
`iam.gke.io/gcp-service-account: k8s-gsm-tools@spiffxp-gke-dev.iam.gserviceaccount.com`

---

Done.

I was following the pattern in `horologium-rbac.yaml` so that I added the double quotes on the name field.
Wonder why they're not working in this case.